students/TracyA/session03/list_lab.py

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
```

Comment:
When you see this in a github file after trying to merge, you need to manually edit that file and remove these kind of statements and any other differences.